addons/website/static/src/xml/website.editor.xml

```diff
@@ -22,6 +22,12 @@
                     '<span class="highlighted-text">#</span>' to link to an anchor.
                 </small>
             </div>
+            <we-row>
+                <we-title>⌙ Media</we-title>
+                <div>
+                    <we-button id="link_upload_file" class="o_we_bg_success" title="Upload file">Upload a file</we-button>
```

Comment:
We don't use `id` but `class`, prefixed by `o_we_` in case of website editor :+1: